web/modules/custom/dpl_search/dpl_search.routing.yml

```diff
@@ -0,0 +1,7 @@
+dpl_search.result:
+  path: '/search'
+  defaults:
+    _controller: '\Drupal\dpl_search\Controller\DplSearchResultController::index'
+    _title: 'DPL Search'
```

Comment:
Consider whether it is helpful and/or required to use a dynamic page title including the query using [`_title_callback`](https://www.drupal.org/docs/drupal-apis/routing-system/structure-of-routes).